client/src/components/App.js

```diff
@@ -26,7 +26,9 @@ const App = (props) => {
     <Router>
       <TopBar user={currentUser} />
       <Switch>
-        <Route exact path="/" component={IndexPage} />
+        <Route exact path="/">
+          <IndexPage user={currentUser} />
```

Comment:
Might want to make this an authenticated route?

Reply:
I do want users that are not signed in to be able to visit this page, I am passing down current user so only signed-in users will be able to save tables